doc/md/getting-started/basic-concepts.md

```diff
@@ -309,150 +137,34 @@ In this case, we can omit this annotation without changing the meaning of the pr
 let x = 1
 ```
 
-Except for some esoteric situations involving operator overloading, type annotations do not (typically) affect the meaning of the program as it runs.
-
-If they are omitted and the compiler accepts the program, as is the case above, the program has the same meaning (same *behavior*) as it did originally.
+Except for some esoteric situations involving operator overloading, type annotations do not typically affect the meaning of the program as it runs. If they are omitted and the compiler accepts the program, as is the case above, the program has the same meaning and behavior as it did originally.
 
-However, sometimes type annotations are required by the compiler to infer other assumptions, and to check the program as a whole.
-
-When they are added and the compiler still accepts the program, we know that the added annotations are *consistent* with the existing ones.
-
-For instance, we can add additional (not required) annotations, and the compiler checks that all annotations and other inferred facts agree as a whole:
-
-``` motoko
-let x : Nat = 1 : Nat
-```
-
-If we were to try to do something *inconsistent* with our annotation type, however, the type checker will signal an error.
-
-Consider this program, which is not well-typed:
-
-``` motoko run
-let x : Text = 1 + 1
-```
-
-The type annotation `Text` does not agree with the rest of the program, since the type of `1 + 1` is `Nat` and not `Text`, and these types are unrelated by subtyping. Consequently, this program is not well-typed, and the compiler will signal an error (with a message and location) and will not compile or execute it.
+However, sometimes type annotations are required by the compiler to infer other assumptions and to check the program as a whole. When they are added and the compiler still accepts the program, you can confirm that the added annotations are consistent with the existing ones.
 
 ## Type errors and messages
 
-Mathematically, the type system of Motoko is *declarative*, meaning that it exists independently of any implementation, as a concept entirely in formal logic. Likewise, the other key aspects of the language definition (for example, its execution semantics) exist outside of an implementation.
-
-However, to design this logical definition, to experiment with it, and to practice making mistakes, we want to interact with this type system, and to make lots of harmless mistakes along the way.
+Mathematically, the type system of Motoko is **declarative**, meaning that it exists independently of any implementation as a concept entirely in formal logic. Likewise, the other key aspects of the language definition, for example its execution semantics, exist outside of an implementation.
```

Comment:
```suggestion
Motoko is a statically typed programming language. That means that he compiler will reject any program that contains obviously non-sensical code. 

For example, while adding two numbers is allowed, adding a number to a text value makes no sense (to Motoko) and is flagged as an error that must be fixed before the code can be run or deployed. 

The rules that Motoko applies to verify the correctness  of code before compiling and running it are called its type system. Motoko's type system will detect and reject static errors such as applying a function to the wrong number of arguments, or to arguments of the wrong type. 

The type system is a safety feature that prevents a slew of errors that would otherwise have to be detected and reported at runtime, when they would be difficult or impossible to address.
```